src/irc/core/irc-servers.h

```diff
@@ -160,7 +160,6 @@ void irc_server_send_action(IRC_SERVER_REC *server, const char *target,
 char **irc_server_split_action(IRC_SERVER_REC *server, const char *target,
 			       const char *data);
 void irc_server_send_away(IRC_SERVER_REC *server, const char *reason);
-void irc_server_send_data(IRC_SERVER_REC *server, const char *data, int len);
```

Comment:
This sounds like removing API

Reply:
I removed that because it was listed under "INTERNAL" (not visible in the diff as it's a couple lines further up). My understanding was that it was listed there only because it had to be called from `src/irc/core/irc.c`.